os_info/src/linux/file_release.rs

```diff
@@ -198,321 +232,301 @@ mod tests {
+                    ..Default::default()
+                }),
+            ),
+            (
+                "src/linux/tests/Pop/22.4.0",
+                Some(Info {
+                    os_type: Type::Pop,
+                    version: Version::Semantic(22, 4, 0),
+                    codename: Some("jammy".to_string()),
+                    ..Default::default()
+                }),
+            ),
+            (
+                "src/linux/tests/Raspbian/10.0.0",
+                Some(Info {
+                    os_type: Type::Raspbian,
+                    version: Version::Semantic(10, 0, 0),
+                    codename: Some("buster".to_owned()),
+                    ..Default::default()
+                }),
+            ),
+            (
+                "src/linux/tests/RedHatEnterprise/8.2.0",
+                Some(Info {
+                    os_type: Type::RedHatEnterprise,
+                    version: Version::Semantic(8, 2, 0),
+                    ..Default::default()
+                }),
+            ),
+            (
+                "src/linux/tests/RedHatEnterprise/7.9.0/server",
+                Some(Info {
+                    os_type: Type::RedHatEnterprise,
+                    version: Version::Semantic(7, 9, 0),
+                    edition: Some("Server".to_owned()),
+                    ..Default::default()
+                }),
+            ),
+            (
+                "src/linux/tests/RedHatEnterprise/7.9.0/server/redhat-release",
+                Some(Info {
+                    os_type: Type::RedHatEnterprise,
+                    version: Version::Semantic(7, 9, 0),
+                    ..Default::default()
+                }),
+            ),
+            (
+                "src/linux/tests/Solus/4.3.0",
+                Some(Info {
+                    os_type: Type::Solus,
+                    version: Version::Semantic(4, 3, 0),
+                    codename: Some("fortitude".to_string()),
+                    ..Default::default()
+                }),
+            ),
+            (
+                "src/linux/tests/SUSE/12.5.0",
+                Some(Info {
+                    os_type: Type::SUSE,
+                    version: Version::Semantic(12, 5, 0),
+                    ..Default::default()
+                }),
+            ),
+            (
+                "src/linux/tests/SUSE/15.2.0",
+                Some(Info {
+                    os_type: Type::SUSE,
+                    version: Version::Semantic(15, 2, 0),
+                    ..Default::default()
+                }),
+            ),
+            (
+                "src/linux/tests/Ubuntu/18.10.0",
+                Some(Info {
+                    os_type: Type::Ubuntu,
+                    version: Version::Semantic(18, 10, 0),
+                    codename: Some("cosmic".to_string()),
+                    ..Default::default()
+                }),
+            ),
+        ];
+
+        for (root, expected) in expected_pairs {
```

Comment:
I also usually prefer to write tests like this, but now it stops at the first failure. Not a big issue, though.